### ISSUE TITLE
BR: Crashes on Startup Java 11 macOS 10.14.0 Version: 2018-09-30 Build

### ISSUE THREAD

**Issue by r-richmond** [dbeaver/dbeaver#4308]
This may be a duplicate of #4208 but I was unable to find an offending line in my .ini file and I have just completed a clean install so I have no old versions of Dbeaver on my system.

Problem: Dbeaver crashes on startup
Message: `See the log file/Users/ryan/.dbeaver4/.metadata/.log.`
Version: I'm attempting to review the changes present in the latest version per #1687 
Log:
```
!SESSION 2018-09-30 15:21:29.595 -----------------------------------------------
eclipse.buildId=unknown
java.version=11
java.vendor=Oracle Corporation
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_US
Framework arguments:  -keyring /Users/user/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/user/.eclipse_keyring

!ENTRY org.jkiss.dbeaver.model 4 0 2018-09-30 15:21:31.038
!MESSAGE Workbench has not been created yet.
!STACK 0
java.lang.IllegalStateException: Workbench has not been created yet.
	at org.eclipse.ui.PlatformUI.getWorkbench(PlatformUI.java:93)
	at org.jkiss.dbeaver.core.application.DBeaverApplication.shutdown(DBeaverApplication.java:290)
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:165)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)

!ENTRY org.eclipse.osgi.compatibility.state.nls 2 0 2018-09-30 15:21:31.087
!MESSAGE Could not resolve module: org.eclipse.osgi.compatibility.state.nls [238]
  Unresolved requirement: Fragment-Host: org.eclipse.osgi.compatibility.state
  Unresolved requirement: Fragment-Host: org.eclipse.osgi.compatibility.state


!ENTRY org.eclipse.e4.ui.workbench.renderers.swt.cocoa.nls 2 0 2018-09-30 15:21:31.087
!MESSAGE Could not resolve module: org.eclipse.e4.ui.workbench.renderers.swt.cocoa.nls [307]
  Unresolved requirement: Fragment-Host: org.eclipse.e4.ui.workbench.renderers.swt.cocoa
  Unresolved requirement: Fragment-Host: org.eclipse.e4.ui.workbench.renderers.swt.cocoa


!ENTRY org.eclipse.core.filesystem.macosx.nls 2 0 2018-09-30 15:21:31.089
!MESSAGE Could not resolve module: org.eclipse.core.filesystem.macosx.nls [293]
  Unresolved requirement: Fragment-Host: org.eclipse.core.filesystem.macosx
  Unresolved requirement: Fragment-Host: org.eclipse.core.filesystem.macosx


!ENTRY org.eclipse.osgi 4 0 2018-09-30 15:21:31.091
!MESSAGE Application error
!STACK 1
org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.IllegalStateException: Workbench has not been created yet.)
	at org.eclipse.swt.SWT.error(SWT.java:4533)
	at org.eclipse.swt.SWT.error(SWT.java:4448)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4034)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3701)
	at org.eclipse.swt.widgets.Display.release(Display.java:3763)
	at org.eclipse.swt.graphics.Device.dispose(Device.java:189)
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:174)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:653)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:590)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1499)
Caused by: java.lang.IllegalStateException: Workbench has not been created yet.
	at org.eclipse.ui.PlatformUI.getWorkbench(PlatformUI.java:93)
	at org.jkiss.dbeaver.core.WorkbenchContextListener.<init>(WorkbenchContextListener.java:57)
	at org.jkiss.dbeaver.core.WorkbenchContextListener.registerInWorkbench(WorkbenchContextListener.java:254)
	at org.jkiss.dbeaver.core.DBeaverUI.lambda$0(DBeaverUI.java:124)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:37)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:182)
	... 17 more

!ENTRY org.eclipse.e4.ui.workbench 4 0 2018-09-30 15:21:31.097
!MESSAGE FrameworkEvent ERROR
!STACK 0
java.lang.NoClassDefFoundError: javax/annotation/PreDestroy
	at org.eclipse.e4.core.internal.di.InjectorImpl.disposed(InjectorImpl.java:450)
	at org.eclipse.e4.core.internal.di.Requestor.disposed(Requestor.java:156)
	at org.eclipse.e4.core.internal.contexts.ContextObjectSupplier$ContextInjectionListener.update(ContextObjectSupplier.java:78)
	at org.eclipse.e4.core.internal.contexts.TrackableComputationExt.update(TrackableComputationExt.java:111)
	at org.eclipse.e4.core.internal.contexts.TrackableComputationExt.handleInvalid(TrackableComputationExt.java:74)
	at org.eclipse.e4.core.internal.contexts.EclipseContext.dispose(EclipseContext.java:178)
	at org.eclipse.e4.core.internal.contexts.osgi.EclipseContextOSGi.dispose(EclipseContextOSGi.java:99)
	at org.eclipse.e4.core.internal.contexts.osgi.EclipseContextOSGi.bundleChanged(EclipseContextOSGi.java:141)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.dispatchEvent(BundleContextImpl.java:908)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:148)
	at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEventPrivileged(EquinoxEventPublisher.java:213)
	at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:120)
	at org.eclipse.osgi.internal.framework.EquinoxEventPublisher.publishBundleEvent(EquinoxEventPublisher.java:112)
	at org.eclipse.osgi.internal.framework.EquinoxContainerAdaptor.publishModuleEvent(EquinoxContainerAdaptor.java:168)
	at org.eclipse.osgi.container.Module.publishEvent(Module.java:476)
	at org.eclipse.osgi.container.Module.doStop(Module.java:634)
	at org.eclipse.osgi.container.Module.stop(Module.java:498)
	at org.eclipse.osgi.container.SystemModule.stop(SystemModule.java:202)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$SystemBundle$EquinoxSystemModule$1.run(EquinoxBundle.java:165)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: javax.annotation.PreDestroy cannot be found by org.eclipse.e4.core.di_1.6.100.v20170421-1418
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:433)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 21 more
```
Dbeaver.ini file
`.../Downloads/DBeaver.app/Contents/Eclipse/dbeaver.ini`
```
-startup
../Eclipse/plugins/org.eclipse.equinox.launcher_1.4.0.v20161219-1356.jar
--launcher.library
../Eclipse/plugins/org.eclipse.equinox.launcher.cocoa.macosx.x86_64_1.1.551.v20171108-1834
-showsplash
-vmargs
-XX:+IgnoreUnrecognizedVMOptions
--add-modules=ALL-SYSTEM
-Xms64m
-Xmx1024m
-XstartOnFirstThread
```

**Comment by serge-rider** [comment]
It is a known issue (Eclipse platform 4.7 isn't compatible with Java 11) - https://bugs.eclipse.org/bugs/show_bug.cgi?id=533390

It will be solved once we'll migrate to Eclipse 4.8 (this Autumn I hope).
Workaround (quite ugly) is to add following lines into the configuration/config.ini file:
```
org.osgi.framework.system.packages = \
 javax.accessibility,\
 javax.activation,\
 javax.activity,\
 javax.crypto,\
 javax.crypto.interfaces,\
 javax.crypto.spec,\
 javax.imageio,\
 javax.imageio.event,\
 javax.imageio.metadata,\
 javax.imageio.plugins.bmp,\
 javax.imageio.plugins.jpeg,\
 javax.imageio.spi,\
 javax.imageio.stream,\
 javax.jws,\
 javax.jws.soap,\
 javax.lang.model,\
 javax.lang.model.element,\
 javax.lang.model.type,\
 javax.lang.model.util,\
 javax.management,\
 javax.management.loading,\
 javax.management.modelmbean,\
 javax.management.monitor,\
 javax.management.openmbean,\
 javax.management.relation,\
 javax.management.remote,\
 javax.management.remote.rmi,\
 javax.management.timer,\
 javax.naming,\
 javax.naming.directory,\
 javax.naming.event,\
 javax.naming.ldap,\
 javax.naming.spi,\
 javax.net,\
 javax.net.ssl,\
 javax.print,\
 javax.print.attribute,\
 javax.print.attribute.standard,\
 javax.print.event,\
 javax.rmi,\
 javax.rmi.CORBA,\
 javax.rmi.ssl,\
 javax.script,\
 javax.security.auth,\
 javax.security.auth.callback,\
 javax.security.auth.kerberos,\
 javax.security.auth.login,\
 javax.security.auth.spi,\
 javax.security.auth.x500,\
 javax.security.cert,\
 javax.security.sasl,\
 javax.sound.midi,\
 javax.sound.midi.spi,\
 javax.sound.sampled,\
 javax.sound.sampled.spi,\
 javax.sql,\
 javax.sql.rowset,\
 javax.sql.rowset.serial,\
 javax.sql.rowset.spi,\
 javax.swing,\
 javax.swing.border,\
 javax.swing.colorchooser,\
 javax.swing.event,\
 javax.swing.filechooser,\
 javax.swing.plaf,\
 javax.swing.plaf.basic,\
 javax.swing.plaf.metal,\
 javax.swing.plaf.multi,\
 javax.swing.plaf.nimbus,\
 javax.swing.plaf.synth,\
 javax.swing.table,\
 javax.swing.text,\
 javax.swing.text.html,\
 javax.swing.text.html.parser,\
 javax.swing.text.rtf,\
 javax.swing.tree,\
 javax.swing.undo,\
 javax.tools,\
 javax.xml,\
 javax.xml.bind,\
 javax.xml.bind.annotation,\
 javax.xml.bind.annotation.adapters,\
 javax.xml.bind.attachment,\
 javax.xml.bind.helpers,\
 javax.xml.bind.util,\
 javax.xml.crypto,\
 javax.xml.crypto.dom,\
 javax.xml.crypto.dsig,\
 javax.xml.crypto.dsig.dom,\
 javax.xml.crypto.dsig.keyinfo,\
 javax.xml.crypto.dsig.spec,\
 javax.xml.datatype,\
 javax.xml.namespace,\
 javax.xml.parsers,\
 javax.xml.soap,\
 javax.xml.stream,\
 javax.xml.stream.events,\
 javax.xml.stream.util,\
 javax.xml.transform,\
 javax.xml.transform.dom,\
 javax.xml.transform.sax,\
 javax.xml.transform.stax,\
 javax.xml.transform.stream,\
 javax.xml.validation,\
 javax.xml.ws,\
 javax.xml.ws.handler,\
 javax.xml.ws.handler.soap,\
 javax.xml.ws.http,\
 javax.xml.ws.soap,\
 javax.xml.ws.spi,\
 javax.xml.ws.spi.http,\
 javax.xml.ws.wsaddressing,\
 javax.xml.xpath,\
 org.ietf.jgss,\
 org.omg.CORBA,\
 org.omg.CORBA_2_3,\
 org.omg.CORBA_2_3.portable,\
 org.omg.CORBA.DynAnyPackage,\
 org.omg.CORBA.ORBPackage,\
 org.omg.CORBA.portable,\
 org.omg.CORBA.TypeCodePackage,\
 org.omg.CosNaming,\
 org.omg.CosNaming.NamingContextExtPackage,\
 org.omg.CosNaming.NamingContextPackage,\
 org.omg.Dynamic,\
 org.omg.DynamicAny,\
 org.omg.DynamicAny.DynAnyFactoryPackage,\
 org.omg.DynamicAny.DynAnyPackage,\
 org.omg.IOP,\
 org.omg.IOP.CodecFactoryPackage,\
 org.omg.IOP.CodecPackage,\
 org.omg.Messaging,\
 org.omg.PortableInterceptor,\
 org.omg.PortableInterceptor.ORBInitInfoPackage,\
 org.omg.PortableServer,\
 org.omg.PortableServer.CurrentPackage,\
 org.omg.PortableServer.POAManagerPackage,\
 org.omg.PortableServer.POAPackage,\
 org.omg.PortableServer.portable,\
 org.omg.PortableServer.ServantLocatorPackage,\
 org.omg.SendingContext,\
 org.omg.stub.java.rmi,\
 org.w3c.dom,\
 org.w3c.dom.bootstrap,\
 org.w3c.dom.css,\
 org.w3c.dom.events,\
 org.w3c.dom.html,\
 org.w3c.dom.ls,\
 org.w3c.dom.ranges,\
 org.w3c.dom.stylesheets,\
 org.w3c.dom.traversal,\
 org.w3c.dom.views,\
 org.w3c.dom.xpath,\
 org.xml.sax,\
 org.xml.sax.ext,\
 org.xml.sax.helpers
```

**Comment by jamesvl** [comment]
Thanks for the above - this helped me also get DBeaver 5.2.1 to run under OpenJDK 11 on Windows 10

**Comment by serge-rider** [comment]
Version 5.2.3 will be based on Eclipse 4.9 platform so this issue should be fixed.

**Comment by firojkabir** [comment]
> It is a known issue (Eclipse platform 4.7 isn't compatible with Java 11) - https://bugs.eclipse.org/bugs/show_bug.cgi?id=533390
> 
> It will be solved once we'll migrate to Eclipse 4.8 (this Autumn I hope).
> Workaround (quite ugly) is to add following lines into the configuration/config.ini file:
> 
> ```
> org.osgi.framework.system.packages = \
>  javax.accessibility,\
>  javax.activation,\
>  javax.activity,\
>  javax.crypto,\
>  javax.crypto.interfaces,\
>  javax.crypto.spec,\
>  javax.imageio,\
>  javax.imageio.event,\
>  javax.imageio.metadata,\
>  javax.imageio.plugins.bmp,\
>  javax.imageio.plugins.jpeg,\
>  javax.imageio.spi,\
>  javax.imageio.stream,\
>  javax.jws,\
>  javax.jws.soap,\
>  javax.lang.model,\
>  javax.lang.model.element,\
>  javax.lang.model.type,\
>  javax.lang.model.util,\
>  javax.management,\
>  javax.management.loading,\
>  javax.management.modelmbean,\
>  javax.management.monitor,\
>  javax.management.openmbean,\
>  javax.management.relation,\
>  javax.management.remote,\
>  javax.management.remote.rmi,\
>  javax.management.timer,\
>  javax.naming,\
>  javax.naming.directory,\
>  javax.naming.event,\
>  javax.naming.ldap,\
>  javax.naming.spi,\
>  javax.net,\
>  javax.net.ssl,\
>  javax.print,\
>  javax.print.attribute,\
>  javax.print.attribute.standard,\
>  javax.print.event,\
>  javax.rmi,\
>  javax.rmi.CORBA,\
>  javax.rmi.ssl,\
>  javax.script,\
>  javax.security.auth,\
>  javax.security.auth.callback,\
>  javax.security.auth.kerberos,\
>  javax.security.auth.login,\
>  javax.security.auth.spi,\
>  javax.security.auth.x500,\
>  javax.security.cert,\
>  javax.security.sasl,\
>  javax.sound.midi,\
>  javax.sound.midi.spi,\
>  javax.sound.sampled,\
>  javax.sound.sampled.spi,\
>  javax.sql,\
>  javax.sql.rowset,\
>  javax.sql.rowset.serial,\
>  javax.sql.rowset.spi,\
>  javax.swing,\
>  javax.swing.border,\
>  javax.swing.colorchooser,\
>  javax.swing.event,\
>  javax.swing.filechooser,\
>  javax.swing.plaf,\
>  javax.swing.plaf.basic,\
>  javax.swing.plaf.metal,\
>  javax.swing.plaf.multi,\
>  javax.swing.plaf.nimbus,\
>  javax.swing.plaf.synth,\
>  javax.swing.table,\
>  javax.swing.text,\
>  javax.swing.text.html,\
>  javax.swing.text.html.parser,\
>  javax.swing.text.rtf,\
>  javax.swing.tree,\
>  javax.swing.undo,\
>  javax.tools,\
>  javax.xml,\
>  javax.xml.bind,\
>  javax.xml.bind.annotation,\
>  javax.xml.bind.annotation.adapters,\
>  javax.xml.bind.attachment,\
>  javax.xml.bind.helpers,\
>  javax.xml.bind.util,\
>  javax.xml.crypto,\
>  javax.xml.crypto.dom,\
>  javax.xml.crypto.dsig,\
>  javax.xml.crypto.dsig.dom,\
>  javax.xml.crypto.dsig.keyinfo,\
>  javax.xml.crypto.dsig.spec,\
>  javax.xml.datatype,\
>  javax.xml.namespace,\
>  javax.xml.parsers,\
>  javax.xml.soap,\
>  javax.xml.stream,\
>  javax.xml.stream.events,\
>  javax.xml.stream.util,\
>  javax.xml.transform,\
>  javax.xml.transform.dom,\
>  javax.xml.transform.sax,\
>  javax.xml.transform.stax,\
>  javax.xml.transform.stream,\
>  javax.xml.validation,\
>  javax.xml.ws,\
>  javax.xml.ws.handler,\
>  javax.xml.ws.handler.soap,\
>  javax.xml.ws.http,\
>  javax.xml.ws.soap,\
>  javax.xml.ws.spi,\
>  javax.xml.ws.spi.http,\
>  javax.xml.ws.wsaddressing,\
>  javax.xml.xpath,\
>  org.ietf.jgss,\
>  org.omg.CORBA,\
>  org.omg.CORBA_2_3,\
>  org.omg.CORBA_2_3.portable,\
>  org.omg.CORBA.DynAnyPackage,\
>  org.omg.CORBA.ORBPackage,\
>  org.omg.CORBA.portable,\
>  org.omg.CORBA.TypeCodePackage,\
>  org.omg.CosNaming,\
>  org.omg.CosNaming.NamingContextExtPackage,\
>  org.omg.CosNaming.NamingContextPackage,\
>  org.omg.Dynamic,\
>  org.omg.DynamicAny,\
>  org.omg.DynamicAny.DynAnyFactoryPackage,\
>  org.omg.DynamicAny.DynAnyPackage,\
>  org.omg.IOP,\
>  org.omg.IOP.CodecFactoryPackage,\
>  org.omg.IOP.CodecPackage,\
>  org.omg.Messaging,\
>  org.omg.PortableInterceptor,\
>  org.omg.PortableInterceptor.ORBInitInfoPackage,\
>  org.omg.PortableServer,\
>  org.omg.PortableServer.CurrentPackage,\
>  org.omg.PortableServer.POAManagerPackage,\
>  org.omg.PortableServer.POAPackage,\
>  org.omg.PortableServer.portable,\
>  org.omg.PortableServer.ServantLocatorPackage,\
>  org.omg.SendingContext,\
>  org.omg.stub.java.rmi,\
>  org.w3c.dom,\
>  org.w3c.dom.bootstrap,\
>  org.w3c.dom.css,\
>  org.w3c.dom.events,\
>  org.w3c.dom.html,\
>  org.w3c.dom.ls,\
>  org.w3c.dom.ranges,\
>  org.w3c.dom.stylesheets,\
>  org.w3c.dom.traversal,\
>  org.w3c.dom.views,\
>  org.w3c.dom.xpath,\
>  org.xml.sax,\
>  org.xml.sax.ext,\
>  org.xml.sax.helpers
> ```

It worked, Thanks a lot.

**Comment by abdul-jabbar01** [comment]
> It is a known issue (Eclipse platform 4.7 isn't compatible with Java 11) - https://bugs.eclipse.org/bugs/show_bug.cgi?id=533390
> 
> It will be solved once we'll migrate to Eclipse 4.8 (this Autumn I hope).
> Workaround (quite ugly) is to add following lines into the configuration/config.ini file:
> 
> ```
> org.osgi.framework.system.packages = \
>  javax.accessibility,\
>  javax.activation,\
>  javax.activity,\
>  javax.crypto,\
>  javax.crypto.interfaces,\
>  javax.crypto.spec,\
>  javax.imageio,\
>  javax.imageio.event,\
>  javax.imageio.metadata,\
>  javax.imageio.plugins.bmp,\
>  javax.imageio.plugins.jpeg,\
>  javax.imageio.spi,\
>  javax.imageio.stream,\
>  javax.jws,\
>  javax.jws.soap,\
>  javax.lang.model,\
>  javax.lang.model.element,\
>  javax.lang.model.type,\
>  javax.lang.model.util,\
>  javax.management,\
>  javax.management.loading,\
>  javax.management.modelmbean,\
>  javax.management.monitor,\
>  javax.management.openmbean,\
>  javax.management.relation,\
>  javax.management.remote,\
>  javax.management.remote.rmi,\
>  javax.management.timer,\
>  javax.naming,\
>  javax.naming.directory,\
>  javax.naming.event,\
>  javax.naming.ldap,\
>  javax.naming.spi,\
>  javax.net,\
>  javax.net.ssl,\
>  javax.print,\
>  javax.print.attribute,\
>  javax.print.attribute.standard,\
>  javax.print.event,\
>  javax.rmi,\
>  javax.rmi.CORBA,\
>  javax.rmi.ssl,\
>  javax.script,\
>  javax.security.auth,\
>  javax.security.auth.callback,\
>  javax.security.auth.kerberos,\
>  javax.security.auth.login,\
>  javax.security.auth.spi,\
>  javax.security.auth.x500,\
>  javax.security.cert,\
>  javax.security.sasl,\
>  javax.sound.midi,\
>  javax.sound.midi.spi,\
>  javax.sound.sampled,\
>  javax.sound.sampled.spi,\
>  javax.sql,\
>  javax.sql.rowset,\
>  javax.sql.rowset.serial,\
>  javax.sql.rowset.spi,\
>  javax.swing,\
>  javax.swing.border,\
>  javax.swing.colorchooser,\
>  javax.swing.event,\
>  javax.swing.filechooser,\
>  javax.swing.plaf,\
>  javax.swing.plaf.basic,\
>  javax.swing.plaf.metal,\
>  javax.swing.plaf.multi,\
>  javax.swing.plaf.nimbus,\
>  javax.swing.plaf.synth,\
>  javax.swing.table,\
>  javax.swing.text,\
>  javax.swing.text.html,\
>  javax.swing.text.html.parser,\
>  javax.swing.text.rtf,\
>  javax.swing.tree,\
>  javax.swing.undo,\
>  javax.tools,\
>  javax.xml,\
>  javax.xml.bind,\
>  javax.xml.bind.annotation,\
>  javax.xml.bind.annotation.adapters,\
>  javax.xml.bind.attachment,\
>  javax.xml.bind.helpers,\
>  javax.xml.bind.util,\
>  javax.xml.crypto,\
>  javax.xml.crypto.dom,\
>  javax.xml.crypto.dsig,\
>  javax.xml.crypto.dsig.dom,\
>  javax.xml.crypto.dsig.keyinfo,\
>  javax.xml.crypto.dsig.spec,\
>  javax.xml.datatype,\
>  javax.xml.namespace,\
>  javax.xml.parsers,\
>  javax.xml.soap,\
>  javax.xml.stream,\
>  javax.xml.stream.events,\
>  javax.xml.stream.util,\
>  javax.xml.transform,\
>  javax.xml.transform.dom,\
>  javax.xml.transform.sax,\
>  javax.xml.transform.stax,\
>  javax.xml.transform.stream,\
>  javax.xml.validation,\
>  javax.xml.ws,\
>  javax.xml.ws.handler,\
>  javax.xml.ws.handler.soap,\
>  javax.xml.ws.http,\
>  javax.xml.ws.soap,\
>  javax.xml.ws.spi,\
>  javax.xml.ws.spi.http,\
>  javax.xml.ws.wsaddressing,\
>  javax.xml.xpath,\
>  org.ietf.jgss,\
>  org.omg.CORBA,\
>  org.omg.CORBA_2_3,\
>  org.omg.CORBA_2_3.portable,\
>  org.omg.CORBA.DynAnyPackage,\
>  org.omg.CORBA.ORBPackage,\
>  org.omg.CORBA.portable,\
>  org.omg.CORBA.TypeCodePackage,\
>  org.omg.CosNaming,\
>  org.omg.CosNaming.NamingContextExtPackage,\
>  org.omg.CosNaming.NamingContextPackage,\
>  org.omg.Dynamic,\
>  org.omg.DynamicAny,\
>  org.omg.DynamicAny.DynAnyFactoryPackage,\
>  org.omg.DynamicAny.DynAnyPackage,\
>  org.omg.IOP,\
>  org.omg.IOP.CodecFactoryPackage,\
>  org.omg.IOP.CodecPackage,\
>  org.omg.Messaging,\
>  org.omg.PortableInterceptor,\
>  org.omg.PortableInterceptor.ORBInitInfoPackage,\
>  org.omg.PortableServer,\
>  org.omg.PortableServer.CurrentPackage,\
>  org.omg.PortableServer.POAManagerPackage,\
>  org.omg.PortableServer.POAPackage,\
>  org.omg.PortableServer.portable,\
>  org.omg.PortableServer.ServantLocatorPackage,\
>  org.omg.SendingContext,\
>  org.omg.stub.java.rmi,\
>  org.w3c.dom,\
>  org.w3c.dom.bootstrap,\
>  org.w3c.dom.css,\
>  org.w3c.dom.events,\
>  org.w3c.dom.html,\
>  org.w3c.dom.ls,\
>  org.w3c.dom.ranges,\
>  org.w3c.dom.stylesheets,\
>  org.w3c.dom.traversal,\
>  org.w3c.dom.views,\
>  org.w3c.dom.xpath,\
>  org.xml.sax,\
>  org.xml.sax.ext,\
>  org.xml.sax.helpers
> ```

It worked for me. In my case, I was having error during installation of `papyrus-rt` tool and I had to add it in `config.ini` file to make it work.
